charts/platform_hub_prereq_helmfile.yaml

```diff
@@ -14,17 +14,17 @@ releases:
       - name: crds.enabled
         value: true
 
-  - name: rcs-deployer
+  - name: rcs-ocm-deployer
     namespace: rcs-deployer-system
     createNamespace: true
     chart: oci://ghcr.io/dana-team/helm-charts/rcs-ocm-deployer
     version: v0.3.1
     disableValidationOnInstall: true
     set:
       - name: config.placementNamespace
-        value: ci-test-ns
+        value: {{ .Values | get "placementNamespace" "" }}
```

Comment:
It is `placementsNamespace` in the rcs chart so this needs to be changed (in the line above).
Probably should also be changed here and in the values for consistency.